Mailer/app/api/v1/Handlers/Mailers/MailTypesByCodeHandler.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Remp\MailerModule\Api\v1\Handlers\Mailers;
+
+use Remp\MailerModule\Api\Validation\InputValidatorTrait;
+use Remp\MailerModule\Repository\ListsRepository;
+use Tomaj\NetteApi\Handlers\BaseHandler;
+use Tomaj\NetteApi\Response\JsonApiResponse;
+
+class MailTypesByCodeHandler extends BaseHandler
+{
+    private $listsRepository;
+    use InputValidatorTrait;
+
+    public function __construct(
+        ListsRepository $listsRepository
+    ) {
+        parent::__construct();
+        $this->listsRepository = $listsRepository;
+    }
+
+    public function handle($payload)
+    {
+        $request = file_get_contents("php://input");
+        $payload = $this->validateInput($request, 'mail-types-by-code.schema.json');
+
+        if ($this->hasErrors()){
+            return $payload;
```

Comment:
Reading this block of code doesn't make any sense: "If there are errors, return payload".. I needed to check what it actually does and spent time understanding the code instead of just reading it.

Please use blocks similar to the original implementation. See e.g. https://github.com/remp2020/crm-users-module/blob/master/src/api/UserMetaUpsertHandler.php#L37